.travis.yml

```diff
@@ -3,7 +3,7 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
+  - 0.7
```

Comment:
Since 0.6 is still supported in the REQUIRE file, 0.6 should be kept in the tests as well. Luckily it seems that it's quite easy to keep supporting 0.6 (see my other comment).

Reply:
Also, there's no need to do this until 0.7 is released, because we also build on `nightly` (see the line below this one).

Additionally, unless we're _dropping support for 0.6_, this line should not be changed. Instead, we could add a third target line `- 0.7` to build on 0.6, 0.7, _and_ 0.8-dev/1.0-dev when 0.7 is released.